tests/robot-tests/tests/libs/common.robot

```diff
@@ -168,6 +168,7 @@ user waits until page contains link
     [Arguments]    ${link_text}   ${wait}=${timeout}
     wait until page contains element  xpath://a[.="${link_text}"]   timeout=${wait}
 
+
```

Comment:
Rogue new line

Reply:
Tried sneaking in there didn't it... terminated.